tests/unittests/core/test_imports.py

```diff
@@ -63,3 +64,34 @@ def test_get_dependency_min_version_spec():
 
     with pytest.raises(PackageNotFoundError, match="invalid"):
         get_dependency_min_version_spec("invalid", "invalid")
+
+
+@requires("torch")
+def my_tensor(i: int):
+    from torch import tensor
+
+    return tensor(i)
```

Comment:
This could just be
```suggestion
    ...
```

What's inside is not important for the test

Reply:
well I may agree on removing return, but we still want to test that import can be there